build.gradle.kts

```diff
@@ -1,11 +1,8 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
-object Versions {
-    const val mapstruct = "1.5.5.Final"
-    const val springdoc = "2.1.0"
-}
 
 plugins {
+
```

Comment:
always try making your PRs clean. 

This line break looks a bit wierd to me

Reply:
Ok , will try to avoid it